Tests/TracingTests/TracedLockTests.swift

```diff
@@ -78,6 +78,12 @@ private final class TracedLockPrintlnTracer: LegacyTracer {
         )
     }
 
+    /// Export all ended spans to the configured backend that have not yet been exported.
+    ///
+    /// This function should only be called in cases where it is absolutely necessary,
+    /// such as when using some FaaS providers that may suspend the process after an invocation, but before the backend exports the completed spans.
+    ///
+    /// This function should not block indefinitely, implementations should offer a configurable timeout for flush operations.
```

Comment:
IMHO not worth putting comments onto types in **Tests** because that's just duplicated content that we'll forget to keep in sync, can we please remove the docs from `TracingTests/...` impls?

Reply:
:facepalm: Didn't intend to do anything for private types, internal types, or tests - just splashed over as I was iterating through "Find..." results. I'll back this out.